Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM fedora
+
+WORKDIR /build
+COPY . .
+
+ENV PY_COLORS=1
+
+RUN dnf -y upgrade openssl-libs
+RUN dnf -y install yum $(cat test-deps-fedora.txt)
```

Comment:
Yes, I strongly agree with your opinion, removing `test-deps-fedora.txt`.
Just in case I kept the file.
As you like removing the file, I will remove the file in this PR.